remote-playback/watch-availability-callback-parameter.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<title>Test that the callback is called with boolean parameter when calling watchAvailability().</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/media.js"></script>
+<script>
+async_test(t => {
+  var v = document.createElement('video');
+  v.src = getVideoURI('/media/movie_5');
+
+  function callback(available) {
+    assert_true(typeof available === "boolean");
+  }
+
+  v.remote.watchAvailability(t.step_func_done(callback)).then(
+    t.step_func(() => {}), t.unreached_func());
```

Comment:
It's legal for the browser to reject watchAvailability at any time.    You can assert that the rejection is [NotSupportedError](https://webidl.spec.whatwg.org/#notsupportederror)